File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/DuelistArmor.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.HeavyArmor;

/* JADX INFO: loaded from: classes3.dex */
public class DuelistArmor extends HeavyArmor {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.armor_heavy_duelist_armor_name;
        this.idDescription = R.string.armor_heavy_duelist_armor_description;
        this.idEffect = R.string.armor_heavy_duelist_armor_effect;
        this.idImage = R.drawable.duelist_armor;
        this.price = 3885L;
        this.maxHp = 75;
        this.constitution = 20;
        this.counterattack = 0.25d;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/DuelistArmor.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.HeavyArmor

class DuelistArmor : HeavyArmor() {
    override fun configureProperties() {
        idName = R.string.armor_heavy_duelist_armor_name
        idDescription = R.string.armor_heavy_duelist_armor_description
        idEffect = R.string.armor_heavy_duelist_armor_effect
        idImage = R.drawable.duelist_armor
        price = 3885L
        maxHp = 75
        constitution = 20
        counterattack = 0.25
    }
}
